.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: required
+language: cpp
+
+services:
+  - docker
+
+before_install:
+  - docker pull boxdot/flatdata-dev
```

Comment:
Would it be possible to use Dockerfile from source tree? Otherwise, who is able to and how to update the image?

Reply:
Do you mean to build the image for each test run? This would take too much time since pandas is compiled inside the docker natively. And, most likely, we want to keep the testing environment stable.

If you mean, to build the image on https://hub.docker.com/ from the Dockerfile automatically. Yes, this is possible, however, I don't have permission to add this repository to my dockerhub account.

Reply:
@boxdot With respect to building image each test run, sounds reasonable. For second question: sorry, I meant, if you suddenly go windsurfing for two weeks, we wouldn't be able to update image at `boxplot/flatdata-dev`, right? Would be cool if there would be `heremaps/flatdata-dev` or otherwise we might want to consider creating `flatdata/dev`. Or am I going overboard?

Reply:
> With respect to building image each test run, sounds reasonable.

Not sure about that. Sounds wasteful, and also dangerous since a failure during the build phase of the image does not necessarily mean that something is wrong with flatdata. Further, an external update of a dependency can break the build. Partially because of that, `Gemfile.lock` (ruby) and `package-lock.json` (npm) were introduced.

> Would be cool if there would be heremaps/flatdata-dev or otherwise we might want to consider creating flatdata/dev. Or am I going overboard?

Not really, I totally agree with your concerns. The best case scenario is when `flatdata-dev` image is built automatically on dockerhub based on the Dockerfile from the source code. A new build should be triggered, when `Dockerfile` is updated in source.

I propose, for now, to go with my intermediate solution. And discuss with `heremaps` a possibility of automatic `heremaps/flatdata-dev` image.

Reply:
> Not sure about that.
I meant your comment sounds reasonable :)
> I propose, for now, to go with my intermediate solution.
As long as this doesn't become permanent solution :) Could you create an issue for us to remember about it?